Source/WebCore/Modules/compression/DecompressionStream.js

```diff
@@ -66,7 +71,13 @@ function initializeDecompressionStream(format)
     };
     const flushAlgorithm = () => {
         const decoder = @getByIdDirectPrivate(this, "DecompressionStreamDecoder");
-        const buffer = decoder.@flush();
+        
+        let buffer;
+        try {
+        buffer = decoder.@flush();
+        } catch (e) {
+            return @Promise.@reject(@makeTypeError(e.message));
+        }
         if (buffer) {
             const transformStream = @getByIdDirectPrivate(this, "DecompressionStreamTransform");
             const controller = @getByIdDirectPrivate(transformStream, "controller");
```

Comment:
Again: Propose you move this stuff inside the 'try' block, since it's never used again.